.circleci/config.yml

```diff
@@ -26,6 +28,18 @@ common: &common
     - run:
         name: install dependencies
         command: pip install --user tox
+    - run:
+        name: check for solc
+        command: |
+          if [[ -n "$SOLC_VERSION" ]]; then 
+            echo "$SOLC_VERSION"
+            sudo python -m pip install "py-solc>=1.2.0"
+            sudo python -m solc.install $SOLC_VERSION
+            echo 'export SOLC_BINARY=/root/.py-solc/solc-$SOLC_VERSION/bin/solc' >> $BASH_ENV
+            source $BASH_ENV
+            sudo chmod u+x $SOLC_BINARY
+            sudo $SOLC_BINARY --version
+          fi
```

Comment:
@pipermerriam @davesque Does anything jump out to you as wrong here? It seems to me like `solc` installs fine - `$SOLC_BINARY` is set correctly for `py-solc` to use - but the test errors out saying `Permission denied` when I try to run `solc.compile`

Reply:
`sudo chmod u+x $SOLC_BINARY` is only giving execution rights to the root user.

Reply:
@carver I also tried `a+rwx` and `-R 755` but neither of those seem to grant the right permissions.